scss/civicrm/_variables.scss

```diff
@@ -27,6 +27,10 @@ $crm-dark-gray-color: #70716b;
 $crm-green-color: #B0D730;
 $crm-light-gray-color: #818c97;
 
+$crm-collapsible-title-background: #42AFCB;
```

Comment:
We do not create any new color variables, we use from pre-defined palette only at https://github.com/civicrm/org.civicrm.shoreditch/blob/develop/scss/bootstrap/overrides/_variables.scss#L33-L51